src/FileManager.php

```diff
@@ -190,6 +190,11 @@ public function getPathForTemplate(string $filename): string
         return $this->twigDefaultPath.'/'.$filename;
     }
 
+    public function removeFile($pathFile): void
+    {
+        $this->fs->remove($pathFile);
+    }
```

Comment:
This needs to output something - see the `dumpFile()` method in this class.

Reply:
Done.